### PR TITLE
Bootstrap

### PR DESCRIPTION
**Pull request by punkrockeng**

### CODE REVIEW

File: thesis-app/src/app/registration/components/signup-stylist-form/signup-stylist-form.component.html [hunk 1]
```diff
@@ -1,44 +1,60 @@
...
+    <div class="input-group">
+      <span class="input-group-addon" id="basic-addon1"><h4>Where are you servicing?</h4></span>
+      <input
+        class="form-control"
+        name="serviceLocation"
+        ngModel
+        style="height: 55px"
+      >
+    </div>
+    <!-- <div class="container row">
```
Comment (Drewclam) [inline thesis-app/src/app/registration/components/signup-stylist-form/signup-stylist-form.component.html:15]: are we taking these out?

File: index.js [hunk 2]
```diff
@@ -12,6 +12,9 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(express.static(path.join(__dirname, 'thesis-app/dist')));
 
+app.get('*', function(req, res) {
```
Comment (Drewclam) [inline index.js]: this looks like a duplicate get route: should be using the one on line 128

File: thesis-app/src/app/registration/components/signup-stylist-form/signup-stylist-form.component.html (hunk 1 above)
Comment (punkrockeng) [reply thesis-app/src/app/registration/components/signup-stylist-form/signup-stylist-form.component.html:15]: I was thinking that we could take them out.

Comment (Drewclam) [reply thesis-app/src/app/registration/components/signup-stylist-form/signup-stylist-form.component.html:15]: Okay, we do not need to store the gender of the stylist?

**Comment by akcays** [comment]
LGTM

**Comment by Drewclam** [comment]
lgtm

**Comment by gitParul** [comment]
lg

**Pull request merged**